Fabric.Identity.API/scripts/identity-cli.psm1

```diff
@@ -0,0 +1,141 @@
+<#
+    .Synopsis
+    Attempts to get an access token for given client
+
+    .Description
+    Takes in a client id and secret with valid scope and hits the identity server provided and returns an access token.
+
+    .Parameter identityUrl
+    The identity url to get the access token from
+
+    .Parameter clientId
+    The id of the client the access token is being requested for
+
+    .Parameter secret
+    The secret of the client the access token is being requested for
+
+    .Parameter scopes
+    The scope of the access token to be requested
+
+    .Example
+    Get-AccessToken -identityUrl "https://server/identity" -clientId "sample-client-id" -secret "SECrEtStrING" -scopes "offline_access"
+#>
+function Get-AccessToken {
+    param(
+        [Parameter(Mandatory=$True)] [Uri] $identityUrl,
+        [Parameter(Mandatory=$True)] [string] $clientId,
+        [Parameter(Mandatory=$True)] [string] $secret,
+        [Parameter(Mandatory=$True)] [string] $scope
+    )
+
+    $url = "$identityUrl/connect/token"
+    $body = @{
+        client_id = "$clientId"
+        grant_type = "client_credentials"
+        scope = "$scope"
+        client_secret = "$secret"
+    }
+
+    $accessTokenResponse = Invoke-RestMethod -Method Post -Uri $url -Body $body
+    return $accessTokenResponse.access_token
+}
+
+<#
+    .Synopsis
+    Attempts to get an access token the fabric installer client
```

Comment:
I think we are missing a word here: "Attempts to get an access token `for` the fabric installer client"